folium/folium.py

```diff
@@ -221,7 +221,9 @@ def __init__(self, location=None, width='100%', height='100%',
         self.template_vars.setdefault('tile_layers', [])
 
     @iter_obj('simple')
-    def add_tile_layer(self, tile_name=None, tile_url=None, active=False):
+    def add_tile_layer(self, tile_name=None, tile_url=None, active=False, minZoom=0,
+                        maxZoom=18, tms=False, continuousWorld=False, noWrap=False, zoomOffset=0,
+                        zoomReverse=False, opacity=1, attribution='Leaflet'):
```

Comment:
`attribution` should be a string about the source of the layer, correct?  If leave the default equal to `None`.